pkg/aututils/aututils.go

```diff
@@ -32,6 +32,40 @@ const (
 	StorageSpecAction = "openstorage.io.action.storagepool/expand"
 )
 
+// PvcRuleByUsageCapacity returns an autopilot pvc expand rule that uses usage of pvc size
+func PvcRuleByUsageCapacity(usage, scalePercentage int, maxSize string) apapi.AutopilotRule {
+	apRuleObject := apapi.AutopilotRule{
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name: fmt.Sprintf("pvc-usage-%d-scale-%d-maxsize-%s", usage, scalePercentage, maxSize),
```

Comment:
add maxsize in the rule name only if user provided it

Reply:
if you use Gi for max size, it would cause the error like
```Failed to create namespace: aut-postgres-volresizeonprem-pvc-total-10Gi  a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character ```